command-expansion-server/src/app.ts

```diff
@@ -333,6 +339,58 @@ app.post('/expand-all-activity-instances', async (req, res, next) => {
   return next();
 });
 
+app.post('/get-seqjson-for-sequence', async (req, res, next) => {
+  // Get the specified sequence + activity instance ids + commands from the latest expansion run for each activity instance (filtered on simulation dataset)
+  // get start time for each activity instance and join with activity instance command data
+  // Create sequence object with all the commands and return the seqjson
+  const context: Context = res.locals.context;
+
+  const seqId = req.body.input.seqId as string;
+  const simulationDatasetId = req.body.input.simulationDatasetId as number;
+
+  const { rows } = await db.query<{
+    metadata: Record<string, unknown>;
+    commands: ReturnType<Command['toSeqJson']>,
+    activity_instance_id: number,
+    errors: string,
+  }>(`
+    select distinct on (activity_instance_commands.expansion_run_id) sequence.metadata, activity_instance_commands.commands, activity_instance_commands.activity_instance_id, activity_instance_commands.errors
```

Comment:
Not necessarily this PR but the formatting for these queries is inconsistent from endpoint to endpoint in this file which might make longterm maintenance harder. Can we switch to use a consistent format? (caps vs lowercase, indentation, etc.)

Reply:
I was thinking about that the whole time I was working on this and had to resist the temptation for refactoring scope creep. When we get the rest of the PR finished up, I'll do a refactoring pass. + commit